main.go

```diff
@@ -97,12 +104,36 @@ func Upload(c *gin.Context) {
 	}
 
 	filename := c.GetHeader("uploader-file-name")
+	chunkid := c.GetHeader("uploader-chunk-number")
+	chunksTotal := c.GetHeader("uploader-chunks-total")
 	path := "./upload/"
 	dst := path + filename //<- destino del archivo
 
 	util.AppendData(dst, fileData)
 
-	c.String(http.StatusOK, "File %s uploaded successfully.", filename)
+	//AWS-Glacier
+	if chunkid == chunksTotal {
```

Comment:
I am not sure I understand this code section, now if we are doing multipart upload we do not need to leave anything on the container unless strictly required, the upload should be piped to Glacier directly if possible, nothing should be left on the OS of the server it self. Please think about this review the code again and tell me your thoughts.